docs/device_report.md

```diff
@@ -70,6 +70,7 @@ Overall device result FAIL
 |skip|connection.port_duplex|Other|Other|No local IP has been set, check system config|
 |skip|connection.port_link|Other|Other|No local IP has been set, check system config|
 |skip|connection.port_speed|Other|Other|No local IP has been set, check system config|
+|pass|manual.test.travis|Security|Recommended|Manual test - for testing|
```

Comment:
This should not explicitly reference "travis" -- does just "manual.test.sample" work?